### ISSUE TITLE
ideia: output standardization

### ISSUE THREAD

**Issue by abgit** [allinurl/goaccess#53]
Hi all,
Currently we have different ways to output content. eg:
```goaccess -a > file.html``` for html
```goaccess -a -o csv > file.csv``` for csv
```goaccess -a -o json > file.json``` for json

What do you think to create a standard format and simplify. eg:
```goaccess -a -o file.html```
```goaccess -a -o file.csv```
```goaccess -a -o file.json```

Standard, simple, no extension duplication (``-o EXT > f.EXT``), no isatty() magic.
If ```-o``` is not used, x-terminal will be displayed

best,
Francisco Azevedo

**Comment by toontong** [comment]
Output tow or more format at the same time is Good for me, like:

> goaccess -a -o report.html -o report.json

or
> goaccess -a -output-html report.html -output-html report.json

then, I can get html and json format at one-run.  My access log file is more then 100G per day.

**Comment by allinurl** [comment]
@toontong v1.0 is taking a different approach on the HTML output, it parses JSON raw data directly so it might be possible to generate directly from the HTML report a csv/json and output that as a blob. See #343 for more details.

**Comment by toontong** [comment]
Thanks, I did'nt see the #343 befor

**Comment by allinurl** [comment]
Implemented and deployed on [version 1.0](https://goaccess.io/). It standardizes the output per issue #53. For instance, it now supports the following.

```
goaccess -f access.log -o /www/file.html
goaccess -f access.log -o /www/file.csv
goaccess -f access.log -o /www/file.json
```

For backwards compatibility, it still supports the following format:

```
goaccess -f access.log > /www/file.html
goaccess -f access.log -o csv > /www/file.csv
goaccess -f access.log -o json > /www/file.json
```
